rmc/utils/constraint.py

```diff
@@ -772,6 +772,10 @@ def get_rescue_1break_transcripts(
     """
     Get transcripts that have a single breakpoint above the 'rescue' threshold.
 
+    This function starts the 'rescue' search by identifying the transcripts
+    that would have been found in the first single breakpoint search with the
+    lower chi square significance threshold.
+
     These transcripts did not have a breakpoint that was over the initial search threshold
     but have a breakpoint that is above the lower 'rescue' search threshold.
```

Comment:
Maybe something along these lines for the docstring (lines 773-784): 
```suggestion
    Get transcripts that have a single breakpoint in the first round of the 'rescue' search.

    These transcripts did not have a single or simultaneous breakpoint that was over 
    the initial search threshold but do have a single breakpoint that is above
    the lower 'rescue' search threshold.

    This function performs the single break search for the first 'rescue' round
    using the statistics already computed during the single break search for the 
    first 'initial' round and writes out the resulting tables:
    a section (transcript)-level breakpoint table and a locus-level table
    for transcripts where a 'rescue' breakpoint was found.
```

Reply:
LGTM!